### ISSUE TITLE
请问能不能补充一下quickstart？

### ISSUE THREAD

**Issue by Jones-match** [CocaVul/Coca#1]

**Comment by Jones-match** [comment]
Could you please add a section "quick start" in the README.md?

**Comment by nguyendacthienngan** [comment]
True, I also need the guideline on how to use the detectors and explainer. Drop this comment for staying in this thread
